coil-base/src/test/java/coil/fetch/HttpFetcherTest.kt

```diff
@@ -51,7 +55,7 @@ class HttpFetcherTest {
         context = ApplicationProvider.getApplicationContext()
         mainDispatcher = createTestMainDispatcher()
         server = createMockWebServer(context, "normal.jpg")
-        cache = Cache(File("build/cache"), 1024).apply { evictAll() }
+        cache = Cache(File("build/cache"), Long.MAX_VALUE).apply { evictAll() }
```

Comment:
Is this change okay?

Reply:
Yep, the cache is evicted at the end of the test.